plots/_plotfuncs.py

```diff
@@ -93,7 +113,7 @@ def get_csd(day: int | str = None, num_days: int = 0, lag: int = 0) -> int:
     return int(day)
 
 
-def _csd_to_utc(csd: int | str, include_time: bool = False) -> str:
+def csd_to_utc(csd: int | str, include_time: bool = False) -> str:
     """Convert a CSD to a formatted UTC day."""
     date = ephemeris.csd_to_unix(int(csd))
```

Comment:
```suggestion
    date = chime_obs.lsd_to_unix(int(csd))
```

Also, above:

Line 107:
```
      return int(ephemeris.chime.get_current_lsd() - num_days - lag)
```
should be changed to
```
      return int(chime_obs.get_current_lsd() - num_days - lag)
```
Line 111:
```
      return int(ephemeris.unix_to_csd(day))
```
should be changed to
```
      return int(chime_obs.unix_to_lsd(day))
```